crates/vizia_core/src/views/stack.rs

```diff
@@ -65,3 +65,21 @@ impl View for ZStack {
         Some("zstack")
     }
 }
+
+/// A view that is used simply to contain other views. Can be useful for style reasons or when working with bindings.
+pub struct Container;
+
+impl Container {
+    pub fn new<F>(cx: &mut Context, content: F) -> Handle<Self>
+    where
+        F: FnOnce(&mut Context),
+    {
+        Self {}.build(cx, content)
+    }
+}
+
+impl View for Container {
+    fn element(&self) -> Option<&'static str> {
+        Some("container")
+    }
+}
```

Comment:
I'm not sure why we need this when we already have `VStack`. The default layout type is column so this is equivalent to a `VStack` but with a different element name. Is it worth adding this just for the different element name?